src/main/java/core/basesyntax/AreaCalculable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculable {
+    double area();
```

Comment:
According to the checklist, method names should be verbs, so consider renaming the method to something like `calculateArea` to clearly represent the action it performs.